invenio_rdm_records/resources/resources.py

```diff
@@ -100,3 +100,48 @@ def search(self):
             identity=g.identity,
         )
         return items.to_dict(), 200
+
+class RDMPidProviderResource(RecordResource):
```

Comment:
minor: Is it needed to inherit from `RecordResource` or can it inherit even higher? like `Resource`?